includes/settings/class-wcpdf-settings-documents.php

```diff
@@ -69,6 +69,16 @@ public function output( $section ) {
 				}
 				?>
 			</ul>
+			<p><i>
+				<?php
+					printf(
+						/* translators: 1. open anchor tag, 2. close anchor tag */
+						__( 'Need more document? Please visit %1$shere%2$s.', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I think this should be behind a Pro check. We don't want to promote this for the free plugin users.

Reply:
What if we inform free users about the Pro plugin documents in the documentation article?
Do you think it would be good to keep this for free users then?

Reply:
Yes, I agree with that.

Reply:
So, is the current text fine?

Reply:
Seems fine to me.